website/server/libs/taskManager.js

```diff
@@ -199,10 +199,19 @@ export async function getTasks (req, res, options = {}) {
   // Order tasks based on tasksOrder
   if (type && type !== 'completedTodos' && type !== '_allCompletedTodos') {
     let order = owner.tasksOrder[type];
-    let orderedTasks = new Array(tasks.length);
+
+    // Remove possible completed todos in the task list
```

Comment:
Given the query here https://github.com/HabitRPG/habitica/pull/8928/files#diff-cb32c22fd9c0bad615b785ed3f988796R182 that excludes completed todos, I don't think it's possible for completed todos to be inside the array